style.css

```diff
@@ -231,141 +231,83 @@ body {
   margin-left: 49%;
 }
 
-/* popup desktop */
+.title-1 {
```

Comment:
- [ ] Good job so far, but as mentioned by the last reviewer your pop-up windows is still cut off the bottom text and information. Please fix it in order to make your project works as expected 👍 

#### Your design
![Capture](https://user-images.githubusercontent.com/6615394/226143640-08ccada7-8173-4f3e-af7c-2a2df304d405.PNG)


#### Figma design
![226142744-efc8cc6c-b1eb-4c70-94b4-8a613a4d604b](https://user-images.githubusercontent.com/6615394/226143586-d7ff049d-6c32-41f5-8c8a-45e31f564385.png)